src/main/java/org/sagebionetworks/web/client/widget/entity/editor/TableQueryResultWikiViewImpl.java

```diff
@@ -17,10 +19,15 @@ public interface TableQueryResultViewUiBinder extends UiBinder<Widget, TableQuer
 	TextBox queryField;
 	@UiField
 	CheckBox isQueryVisible;
+	@UiField
+	CheckBox isShowTableOnly;
 
 	@Inject
-	public TableQueryResultWikiViewImpl(TableQueryResultViewUiBinder binder) {
+	public TableQueryResultWikiViewImpl(TableQueryResultViewUiBinder binder, CookieProvider cookies) {
 		widget = binder.createAndBindUi(this);
+		boolean isInTestWebsiteMode = DisplayUtils.isInTestWebsite(cookies);
+		isQueryVisible.setVisible(!isInTestWebsiteMode);
+		isShowTableOnly.setVisible(isInTestWebsiteMode);
```

Comment:
isShowTableOnly is in experimental mode only (supported by the new table entity widget).